utils/python/CIME/check_input_data.py

```diff
@@ -160,7 +160,7 @@ def check_input_data(case, svn_loc=None, input_data_root=None, data_list_dir="Bu
                                 logging.warning("    Cannot download file since it lives outside of the input_data_root '%s'" % input_data_root)
                             no_files_missing = False
                         else:
-                            logging.info("  Found input file: '%s'" % full_path)
+                            logging.debug("  Found input file: '%s'" % full_path)
```

Comment:
I found it a bit overwhelming, now it tells you each list it's checking then tells you if it can't find anything in the list.   Also changed the case.build output so that it shows you the log it's writing to.